src/app/directives/sidebar.js

```diff
@@ -84,8 +84,8 @@
           var editorHeight = 50;
 
           if (jqXhr.responseText) {
```

Comment:
Why is this check still here if then the `jqXhr.responseText` is not used anywhere?